mussel

```diff
@@ -205,6 +207,12 @@ nettransfer() {
   unset fname nproc url
 }
 
+# ----- Print help for command line arguments ----- #
+help() {
```

Comment:
@firasuke A small refactor; I know this could've gone into another branch, but I decided to do it since I was already here and the repetition was annoying me a little.

Reply:
Sure, refactoring is welcome. I think it is better to try and not break the 1000 LOC for mussel for simplicity purposes, unless there's really a desperate need to do so..

Reply:
Theoretically this has cut 4 lines or so per not repeating the same code many times.
I will check if it still 1000 LOC.

Reply:
Yep, 990 LOC without counting comments nor blanks.